### PR TITLE
[Sage-8] Tokens - Font Family Update to Circular

### PR DESCRIPTION
**Pull request by anechol**
## Description

Updates the Sage font family from Inter to Circular.

Note: Linting is failing because the fonts aren't being committed.

## Screenshots

|  Before  |  After  |
|--------|--------|
Screen Shot 2022-01-27 at 11 19 04 AM|Screen Shot 2022-01-27 at 11 18 28 AM

## Testing in `sage-lib`

- View Sage Docs
- Check that new font, Circular, is properly in use.

## Testing in `kajabi-products`

(LOW) Updates the Sage font family from Inter to Circular. No effect on Kajabi Products.

## Related

[Sage-8](https://kajabi.atlassian.net/browse/SAGE-8)

### CODE REVIEW

File: packages/sage-assets/lib/stylesheets/core/_fonts.scss [hunk 1]
```diff
@@ -4,86 +4,80 @@
...
   font-weight: 400;
   font-display: swap;
-  src: local("Inter-Regular"), url("#{$-body-font-path}/Inter-Regular.woff2?v=#{$-body-font-version}") format("woff2"), url("#{$-body-font-path}/Inter-Regular.woff?v=#{$-body-font-version}") format("woff");
+  src: local("#{$-body-font-name}-Regular"), url("#{$-body-font-path}/#{$-body-font-name}-Regular.woff2?v=#{$-body-font-version}") format("woff2"), url("#{$-body-font-path}/#{$-body-font-name}-Regular.woff?v=#{$-body-font-version}") format("woff");
 }
 
+// Regular Italic
 @font-face {
-  font-family: "Inter";
+  font-family: "CircularXXWeb";
```
Comment (philschanely) [inline packages/sage-assets/lib/stylesheets/core/_fonts.scss:23]: What do you think about shortening this to just "Circular"?

**Review by philschanely — COMMENTED** [review]
Looking good!

File: packages/sage-assets/lib/stylesheets/core/_fonts.scss (hunk 1 above)
Comment (anechol) [reply packages/sage-assets/lib/stylesheets/core/_fonts.scss:23]: Yeah I thought about that, but I was wondering if that would have been in violation of the license of changing the name. Thoughts? 🤔 I might be overthinking here haha

File: .gitignore [hunk 2]
```diff
@@ -10,3 +10,5 @@ lerna-debug.log
 .DS_Store
 coverage
 package-lock.json
+packages/sage-assets/lib/fonts/circular/*
+!packages/sage-assets/lib/fonts/circular/.gitkeep
```
Comment (monicawheeler) [inline .gitignore:14]: Minor: end of line needed here

Comment (anechol) [reply .gitignore]: Appreciate it!

**Review by philschanely — APPROVED** [review]
🙌

**Comment by philschanely** [comment]
Regarding the failing lint we have options:

- Merge into our epic branch and know we'll fix it shortly
- Hold this PR for a bit as we work out the last bits of the CDN-based approach to the font paths.

**Review by monicawheeler — APPROVED** [review]

**Comment by anechol** [comment]
> Regarding the failing lint we have options:
> 
> * Merge into our epic branch and know we'll fix it shortly
> * Hold this PR for a bit as we work out the last bits of the CDN-based approach to the font paths.

I can hold on to this PR and update when you're ready.

**Review by QuintonJason — APPROVED** [review]
Looks and works great

**Comment by philschanely** [comment]
@anechol we're set up and ready with all the fonts under `https://sage.kajabi-cdn.com/fonts/circular`

**Comment by philschanely** [comment]
I'm doing a separate PR on `develop` to handle this for the existing fonts FWIW.

File: packages/sage-assets/lib/stylesheets/core/_fonts.scss (hunk 1 above)
Comment (philschanely) [reply packages/sage-assets/lib/stylesheets/core/_fonts.scss]: Oo good point. On account of the foundry refering to it as Circular I think we're ok

https://lineto.com/typefaces/circular

**Comment by philschanely** [comment]
For reference as needed https://github.com/Kajabi/sage-lib/pull/1183

**Pull request merged**